# Methylation Calling and Tumor Classification from cf-DNA of Pediatric and Adult Cancer types

**Design Project Group 6**  2019 - 2020

Lore Depuydt, Jannes Dewitte, Maísa Santos and Ke Zhang



**Prof.** Jan Fostier

***Supervisors:*** Prof. Dr. Katleen De Preter, Dr. Ruben Van Paemel, Dr. Jilke De Wilde



# **Index**

[**Introduction**](#introduction)

[**Materials and Methods** ](#materials-and-methods)

[*A. GitHub* ](#a.-github)

[*B. Snakemake*](#b.-snakemake)

[*C. Bismark Pipeline* ](#c.-pipeline)

[*D. HPC* ](#d.-hpc)

[**Result and Conclusion**](#result-and-conclusion)

[**References** ](#references)

[**Appendix** ](#appendix)

# **Introduction**

The idea behind this project started from a very common problem: in many cases it is not possible to perform a normal surgical biopsy, because the procedure is too risky, painful and causes anxiety to patients or because in some types of tumours, sampling for diagnosis is not possible (for example, in the case of very big ones that might rupture and cause the spread of tumour cells). Therefore, the solution was to conduct liquid ones.

When cells die, their genetic material is released in the blood stream and can be picked up in blood plasma or on cerebrospinal fluid, making this the principal behind the liquid biopsies and why it was considered a solution to the classical ones. This medical test can be performed in really small quantities of cell free DNA, which is very good since it is usually not possible to obtain large samples from children or from the deteriorated genetic material in paraffin embed tissues (the clinical application in this case is related to CUPs, that are currently 5% of cancer diagnosis; in this cases, since the origin can't be determined due to metastasis, it's difficult to decide what treatment to apply).

The cell free DNA isolated is treated with sodium bisulfate. This will cause the conversion of the unmethylated cytosines to uracil and later to thymine during the PCR. DNA methylation is tissue and cancer specific, so the profile obtained is then compared to reference sets of known tumour tissues and of healthy controls.

# **Materials and Methods**

### *A. GitHub*

### *B. Snakemake*

### *C. Pipeline*

Bismark is a set of tools used for analyzing Bisulfite-Seq (BS-Seq) data in a time-efficient way. It performs mapping bisulfite converted sequence reads to a genome of interested and cytosine methylation calling in one step. It supports both single-end (SE) and paired-end (PE) read alignments, as well as gapped and un-gapped alignments. The output determines cytosine methylation in different states: CpG, CHG and CHH context.

The language used is Perl. Bismark is run from the command line. Read aligner tools for aligning sequencing reads to long reference sequences such as *Bowtie2* or *HISAT2* are required. Additionally, *Samtools* are used for manipulating alignments like sorting, merging, indexing, generating alignments in a single position formation in the BAM format.

![](https://i.imgur.com/wT91MiV.png)


-   *Pipeline Steps*

> **Demultiplex**: to demultiplex a bcl file into a fastq file, which can be further used in the pipeline by using the *bcl2fastq* tool.
>
> **Adaptor trimming:** trim off the adaptors from sequencing data, PE and SE flags should be specified. The trimming software used is *Trim Galore*.
>
> **Mapping\_bismark:** to map the now trimmed sequences to a reference genome.
>
> **Sort\_samtools:** to sort and index the a BAM file. These steps will be performed in 2 different shell command lines.
>
> **Hsmetrics:** Before the hsmetrics rule will be run, a valid genome build should be specified. The hsmetrics rule will collect metrics data for the sequencing data obtained after hybrid selection, which involves the use baits to select desired fragments.
>
> **Mapping\_to\_lambda\_bismark:** to use the files generated by the adaptortrimming rule, so when looking at the pipeline, this rule branches of after adaptortrimming. This time, mapping will be done to the lambda phage genome.
>
> **Extract\_methylation\_calls:** This rule will extract the methylation call for every single C analyzed. The position of every single C will be written out to a new output file, depending on its context (CpG, CHG or CHH).
>
> **Deduplicate:** The script deduplicate\_bismark {input} is used to remove alignments to the same position in the genome from the Bismark mapping output, which can arise by e.g. excessive PCR amplification. Sequences which align to the same genomic position but on different strands are scored individually.
>
> **Extract\_methylation\_calls\_deduplicated:** This rule is exactly the same as in extract\_methylation\_calls, but the input will be the deduplicated reads found in the output from the \'deduplicate\' rule.
>
> **Rename:** The rename rule is used to alter the names of some of the output files generated in previous rules in order to make them shorter.
>
> **Multiqc:** Here a multiqc report will be generated by using multiqc. A report containing the hsmetrics '.txt' file and the coverage files for both human genome (deduplicated) and lambda genome will be made.

### *D. HPC*

# **Result and Conclusion**

# **References**

\[1\] Paemel, R. Van et al. *"Minimally invasive classification of pediatric solid tumors using reduced representation bisulfite sequencing of cell-free DNA: a proof-of-principle study"* (2019). doi: 10.1101/795047.

\[2\] Köster, Johannes and Rahmann, Sven. "Snakemake - A scalable bioinformatics workflow engine". Bioinformatics 2012.

\[3\] Krueger, Felix, and Simon R. Andrews. \"Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications.\" bioinformatics 27.11 (2011): 1571-1572.

# **Appendix**